assembly/assembly-ide-war/pom.xml

```diff
@@ -287,6 +287,24 @@
             <artifactId>plugin-terminal-ui</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.eclipse.lsp4j</groupId>
+            <artifactId>org.eclipse.lsp4j</artifactId>
+            <exclusions>
```

Comment:
No, the exclusion is specific to the client side: we need to include a different xtext.xbase library for gwt than for regular Java. Moving the exclusion to che-dependencies would make it global, no?